relayer/src/foreign_client.rs

```diff
@@ -1069,6 +1062,7 @@ impl<DstChain: ChainHandle, SrcChain: ChainHandle> ForeignClient<DstChain, SrcCh
     /// Retrieves all consensus heights for this client sorted in descending
     /// order.
     fn consensus_state_heights(&self) -> Result<Vec<Height>, ForeignClientError> {
+        // [TODO] Utilize query that only fetches consensus state heights
```

Comment:
Maybe we can even tag the corresponding issue here, so we know in future when we get unblocked to work on this TODO.

```suggestion
        // [TODO] Utilize query that only fetches consensus state heights
        // https://github.com/cosmos/ibc-go/issues/798
```